config/config-network.yaml

```diff
@@ -34,7 +34,8 @@ data:
     # to users that `kubectl edit` this config map.
     #
     # These sample configuration options may be copied out of
-    # this block and unindented to actually change the configuration.
+    # this example block and unindented to be in the data block
+    # to actually change the configuration.
```

Comment:
I think this applies to all YAMLs now. Care to change that everywhere?

Reply:
oh wow, I didn't realize there were more of these :) yep, I'll update them all.